test/test614.js

```diff
@@ -0,0 +1,28 @@
+if (typeof exports === 'object') {
+    var assert = require("assert");
+    var alasql = require('..');
+}
+
+var test = '614'; // insert test file number
+
+describe('Test ' + test + ' - Read data from column irrespective of case in query', function () {
+
+    before(function () {
+        alasql('create database test' + test);
+        alasql('use test' + test);
+        alasql.options.casesensitive = false;
+    });
+
+    after(function () {
+        alasql('drop database test' + test);
+        alasql.options.casesensitive = true;
+    });
+
+    it('1) Read Data and Check Value', function (done) {
+        alasql('SELECT Account FROM XLSX("' + __dirname + '/test614.xlsx")', [], function (res) {
+            assert.equal(res[0]['Account'.toLowerCase()], 12);
+            done();
+        });
+    });
```

Comment:
I suggest having a few more tests both from a positive and a negative side.

A negative test would be that setting the flag to false, the same test would give an error / not find data

there could also be two columns with different case'ing setup (one only upper, one only lower) where it could be tested what happens on casesensetive = off or on